src/microbe_stage/organelle_components/MovementComponent.cs

```diff
@@ -101,7 +101,10 @@ private Vector3 CalculateMovementForce(Microbe microbe, float elapsed)
         // The movementDirection is the player or AI input
         Vector3 direction = microbe.MovementDirection;
 
-        var forceMagnitude = force.Dot(direction);
+        // Real force the flagella applied to the colony (considering rotation)
+        var realForce = organelle.RotatedPositionInsideColony(force);
```

Comment:
I think to save on performance it would be nice to have an if guarding this something like:
```
if(is inside colony)
    force = organelle.RotatedPositionInsideColony(force);
```

Reply:
Actually reading the other parts of the code, I think `RotatedPositionInsideColony` should have an early exit if the microbe is not in a colony to save on some performance when spawning cells with a lot of organelles as well as when a cell with a lot of movement organelles is trying to move (while not being in a colony).